xbmc/GUIPassword.cpp

```diff
@@ -314,9 +313,8 @@ bool CGUIPassword::CheckLock(LockType btnType, const std::string& strPassword, i
       CProfilesManager::GetInstance().GetMasterProfile().getLockMode() == LOCK_MODE_EVERYONE || g_passwordManager.bMasterUser)
     return true;
 
-  int iVerifyPasswordResult = -1;
   std::string strHeading = g_localizeStrings.Get(iHeading);
-  iVerifyPasswordResult = VerifyPassword(btnType, strPassword, strHeading);
+  int iVerifyPasswordResult = VerifyPassword(btnType, strPassword, strHeading);
```

Comment:
It doesn't matter, I think. The variable in CGUIPassword::VerifyPassword gets set to a different value in the "case" statements (depending on the return value of the functions being called), and if no statement matches, the default statement sets it to 0.